cer-graphql/index.js

```diff
@@ -8,14 +8,20 @@ const jwkToPem = require('jwk-to-pem');
 // Measure server startup time
 var startTime = new Date().getTime();
 
-// Contentful settings
-const CONTENTFUL_ACCESS_TOKEN = process.env.CONTENTFUL_ACCESS_TOKEN;
-const CONTENTFUL_SPACE_ID = process.env.CONTENTFUL_SPACE_ID;
-const COGNITO_USER_POOL = process.env.COGNITO_USER_POOL;
-const COGNITO_REGION = process.env.COGNITO_REGION;
-
-// Cognito public key URL
-const COGNITO_PUBLIC_KEYS_URL = `https://cognito-idp.${COGNITO_REGION}.amazonaws.com/${COGNITO_USER_POOL}/.well-known/jwks.json`;
+const getCredentials = (isFromFile) => {
+    if (isFromFile) {
+        const configResult = require('dotenv').config();
+        if (configResult.error){
+            throw configResult.error;
+        }
+    }
+    return {
```

Comment:
Maybe throw a comment here explaining that if `!isFromFile` it returns from your local env (not that we wanna encourage that)?

Reply:
:+1: Comment added to explain what isFromFile does.